go/tools/builders/protoc.go

```diff
@@ -27,21 +28,32 @@ import (
 
 func run(args []string) error {
 	// process the args
+	options := multiFlag{}
+	descriptors := multiFlag{}
 	expected := multiFlag{}
+	imports := multiFlag{}
 	flags := flag.NewFlagSet("protoc", flag.ExitOnError)
 	protoc := flags.String("protoc", "", "The path to the real protoc.")
-	descriptor_set_in := flags.String("descriptor_set_in", "", "The descriptor set to read.")
-	go_out := flags.String("go_out", "", "The go plugin options.")
+	outPath := flags.String("out_path", "", "The base output path to write to.")
 	plugin := flags.String("plugin", "", "The go plugin to use.")
 	importpath := flags.String("importpath", "", "The importpath for the generated sources.")
+	flags.Var(&options, "option", "The plugin options.")
+	flags.Var(&descriptors, "descriptor_set", "The descriptor set to read.")
 	flags.Var(&expected, "expected", "The expected output files.")
+	flags.Var(&imports, "import", "Map a proto file to an import path.")
 	if err := flags.Parse(args); err != nil {
 		return err
 	}
+	plugin_base := filepath.Base(*plugin)
```

Comment:
Nit: `pluginBase` and `pluginName`

Reply:
That's what happens when you try to write skylark and go at the same time...